playbooks/roles/edx_django_service/tasks/main.yml

```diff
@@ -213,7 +213,9 @@
     - install:app-requirements
 
 - name: "Install Datadog APM requirements"
-  when: COMMON_ENABLE_DATADOG and COMMON_ENABLE_DATADOG_APP
+  when: >
+    COMMON_ENABLE_DATADOG and
+    (COMMON_ENABLE_DATADOG_APP or edx_django_service_name in edx_specifically_enabled_datadog_apps)
   pip:
```

Comment:
i'm no ansible expert, so, disclosure that i stole syntax from: https://github.com/edx/configuration/blob/01d773fbd5efaa0cd4616093794d1b8a25b84e65/playbooks/roles/user/tasks/main.yml#L261